sonar-project.properties

```
@@ -12,20 +12,20 @@ sonar.issues.defaultAssigneeLogin=LayZeeDK@github
 sonar.sourceEncoding=UTF-8
 
 # comma-separated paths to directories containing source files
-sonar.sources=apps,libs,tools
+sonar.sources=e2e,packages,tools
 # configure the files that should be completely ignored by the analysis
 sonar.exclusions=**/*.spec.ts
 
 # comma-separated paths to directories containing tests
-sonar.tests=apps,libs,tools
+sonar.tests=e2e,packages,tools
 # patterns used to include some test files and only these ones in analysis
 sonar.test.inclusions=**/*.spec.ts
 
 # configure the files that should be ignored by code coverage calculations
-sonar.coverage.exclusions=tools/scripts/**,libs/internal/**,**/jest.config.js,**/test.ts,apps/**/main.ts,apps/**/polyfills.ts
+sonar.coverage.exclusions=tools/scripts/**,packages/internal/**,**/jest.config.js,**/test.ts,e2e/**/main.ts,e2e/**/polyfills.ts
 
 # configure the files that should be ignored by duplication detection
-sonar.cpd.exclusions=**/jest.config.js,**/test.ts,apps/**/main.ts,apps/**/polyfills.ts
+sonar.cpd.exclusions=**/jest.config.js,**/test.ts,e2e/**/main.ts,e2e/**/polyfills.ts
```

issue: Since we decided that application projects should reside in `packages`, not `e2e`, the `main.ts` and `polyfills.ts` patterns must be updated.
```suggestion
sonar.cpd.exclusions=**/jest.config.js,**/test.ts,packages/**/main.ts,packages/**/polyfills.ts
```

---

This is still an issue.
```suggestion
sonar.cpd.exclusions=**/jest.config.js,**/test.ts,packages/**/main.ts,packages/**/polyfills.ts
```